command/agent/local_test.go

```diff
@@ -194,14 +201,17 @@ func TestAgentAntiEntropy_Services(t *testing.T) {
 	// Trigger anti-entropy run and wait
 	agent.StartSync()
 
-	verifyServicesAfterRemove := func() (bool, error) {
+retry2:
```

Comment:
This ended up being a little gross but that's more an indicator that this test is too nested vs. an issue with the retry-er. I wouldn't change it now, but it's good that this encourages less complicated loops vs. deeply nested returns from the old pattern.